<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IEC61850 Simulator v1.0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="modeFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="modeLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="modeLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>é€‰æ‹©æ¨¡å¼:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer1">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="serverModeBtn">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>ä»¿çœŸIEDè®¾å¤‡ï¼Œä½œä¸ºMMSæœåŠ¡å™¨è¿è¡Œ</string>
         </property>
         <property name="text">
          <string>ğŸ–¥ï¸ æœåŠ¡ç«¯æ¨¡å¼</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clientModeBtn">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>è¿æ¥åˆ°IEDè®¾å¤‡ï¼Œè¯»å†™æ•°æ®ç‚¹</string>
         </property>
         <property name="text">
          <string>ğŸ’» å®¢æˆ·ç«¯æ¨¡å¼</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="modeDescLabel">
         <property name="styleSheet">
          <string notr="true">color: #666; font-style: italic;</string>
         </property>
         <property name="text">
          <string>ä½œä¸ºIEDä»¿çœŸå™¨è¿è¡Œï¼Œç­‰å¾…å®¢æˆ·ç«¯è¿æ¥</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QStackedWidget" name="panelStack">
       <property name="currentIndex">
        <number>0</number>
       </property>
      </widget>
      <widget class="QWidget" name="logWidgetContainer" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>æ–‡ä»¶(&amp;F)</string>
    </property>
    <addaction name="actionLoadConfig"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>è§†å›¾(&amp;V)</string>
    </property>
    <addaction name="actionShowLog"/>
    <addaction name="actionClearLog"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>å¸®åŠ©(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>ä¸»å·¥å…·æ </string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionLoadConfig">
   <property name="text">
    <string>åŠ è½½é…ç½®(&amp;L)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>ä¿å­˜é…ç½®(&amp;S)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>é€€å‡º(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>æ˜¾ç¤ºæ—¥å¿—é¢æ¿(&amp;L)</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="text">
    <string>æ¸…é™¤æ—¥å¿—(&amp;C)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>å…³äº(&amp;A)...</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>â–¶ å¯åŠ¨</string>
   </property>
   <property name="toolTip">
    <string>å¯åŠ¨æœåŠ¡/è¿æ¥</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>â¹ åœæ­¢</string>
   </property>
   <property name="toolTip">
    <string>åœæ­¢æœåŠ¡/æ–­å¼€è¿æ¥</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>ğŸ”„ åˆ·æ–°</string>
   </property>
   <property name="toolTip">
    <string>åˆ·æ–°æ•°æ®</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
